otter/test/test_auth.py

```diff
@@ -827,12 +856,11 @@ def test_composition(self):
         self.assertEqual(ra._max_retries, 3)
         self.assertEqual(ra._retry_interval, 5)
 
-        ia = ra._authenticator
-        self.assertIsInstance(ia, ImpersonatingAuthenticator)
-        self.assertEqual(ia._identity_admin_user, 'uname')
-        self.assertEqual(ia._identity_admin_password, 'pwd')
-        self.assertEqual(ia._url, 'htp')
-        self.assertEqual(ia._admin_url, 'ad')
+        st = ra._authenticator
+        self.assertIsInstance(st, SingleTenantAuthenticator)
```

Comment:
This is the composition test which verifies the authenticator is generated correctly. Shouldn't the the single tenant authenticator be the one used?